mod/wikirate/set/right/my_contributions.rb

```diff
@@ -1,4 +1,37 @@
+def virtual?
+  true
+end
+
+def sections
+  @sections ||= begin
+    user_card = left
+    [ {:name=>'Metrics',  :contributions => :contributed_metrics},
+      {:name=>'Claims',   :contributions => :contributed_claims},
+      {:name=>'Sources',  :contributions => :contributed_sources},
+      {:name=>'Articles', :contributions => :contributed_analysis},
+      {:name=>'Campaigns',:contributions => :contributed_campaigns}
+    ].map do |args|
+      c_card     = user_card.fetch(:trait=>args[:contributions])
+      count      = c_card && c_card.contribution_count
+      contr_name = c_card && c_card.cardname.url_key
+      [ (count || 0), args[:name], contr_name ]
+    end
+  end
+end
```

Comment:
... and less repetitive (had the same searches here as in the "contributed_..." cards)